geode-apis-compatible-with-redis/src/acceptanceTest/resources/0001-configure-redis-tests.patch

```diff
@@ -1,2786 +0,0 @@
-diff --git a/tests/support/server.tcl b/tests/support/server.tcl
```

Comment:
I think this file was accidentally removed.

Reply:
Oy. That's weird. I'll get it back in.